build_wheel.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+#************************************************#
+#           build_wheel.sh                       #
```

Comment:
Your mileage may vary, but in my experience, duplicating the file name within the file adds little value and adds a solid risk of the name here getting out of sync with the name of the file.